recipes/lib-detect-testenv/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This file is in the tarball with the name `LICENSE`, no `.txt`, please remove it here and let conda use the one in the package instead.